File: @LV1/1.4/numberfun/numberfun.py
def solveable(a, b, c):
    if a + b == c or a * b == c:
        return True
    if abs(a - b) == c:
        return True
    if a / b == c or b / a == c:
        return True
    return False


n = int(input())

lines = [[int(i) for i in input().split(" ")] for _ in range(n)]
ans = ["Possible" if solveable(*line) else "Impossible" for line in lines]

[print(n) for n in ans]

File: @LV1/1.4/numberfun/numberfun.rs
use std::io::{self, BufRead};

fn solvable(a: f64, b: f64, c: f64) -> bool {
    if a + b == c || a * b == c {
        return true;
    } else if a - b == c || b - a == c {
        return true;
    } else if a / b == c || b / a == c {
        return true;
    }
    return false;
}

fn main() {
    let stdin = io::stdin();
    let mut buff = stdin.lock().lines().map(|l| l.unwrap());
    buff.next();

    for line in buff {
        let num: Vec<f64> = line
            .split_whitespace()
            .map(|n| n.parse().unwrap())
            .collect();

        if solvable(num[0], num[1], num[2]) {
            println!("Possible");
        } else {
            println!("Impossible");
        }
    }
}
